### PR TITLE
:recycle: Accept-Language header translations

### LINKED ISSUE #259: [translations] Investigate if we can use Accept-Language header to serve en/sv
## Short description of the feature
Instead of having two fields in the graphql schema, send the correct translation based on the Accept-Language header

[X] Relates to frontend
[X] Relates to the backend
[X] Requires modification of GraphQL
[ ] Requires modification of Database
[ ] Requires creating a new service

## Long description with technical specifications
This is apparently the "correct" way to handle translations, and seems really nice since no logic is required on the frontend and new languanges are really easy to add.

### PR DESCRIPTION
**Pull request by emilwihlander**
closes #259

### CODE REVIEW

File: frontend/pages/news/article/[id]/edit.tsx [hunk 1]
```diff
@@ -71,6 +71,10 @@ export default function EditArticlePage() {
       fileType = await FileType.fromBlob(imageFile);
       setImageName(`public/${uuidv4()}.${fileType.ext}`);
     }
+    console.log(queryData);
```
Comment (oliverlevay) [inline frontend/pages/news/article/[id]/edit.tsx:74]: lots of console logs that need to go

**Review by oliverlevay — CHANGES_REQUESTED** [review]
This is AWESOME! It really makes the gql queries much cleaner. There are some changes I'd like you to make.

**Comment by oliverlevay** [comment]
I love this PR ! <3 ... but it's hopelessly outdated. The fastest to fix this would proboably be to take inspiration from this code and recreate it. @emilwihlander

**Comment by danieladugyan** [comment]
> I love this PR ! <3 ... but it's hopelessly outdated. The fastest to fix this would proboably be to take inspiration from this code and recreate it. @emilwihlander

**Pull request closed without merging**